src/drivers/device/integrator.cpp

```diff
@@ -160,9 +172,15 @@ Integrator::get_and_filtered(bool reset, uint64_t &integral_dt, math::Vector<3>
 void
 Integrator::_reset(uint64_t &integral_dt)
 {
-	_integral(0) = 0.0f;
-	_integral(1) = 0.0f;
-	_integral(2) = 0.0f;
+	_alpha(0) = 0.0f;
+	_alpha(1) = 0.0f;
+	_alpha(2) = 0.0f;
+	_last_alpha(0) = 0.0f;
+	_last_alpha(1) = 0.0f;
+	_last_alpha(2) = 0.0f;
+	_beta(0) = 0.0f;
+	_beta(1) = 0.0f;
+	_beta(2) = 0.0f;
```

Comment:
Is it correct that `_last_val` is not reset but the last value from the previous integration is used?